src/main/java/ladder/domain/LadderGame.java

```diff
@@ -0,0 +1,47 @@
+package ladder.domain;
+
+import ladder.util.InputUtil;
+import ladder.view.PrintResult;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class LadderGame {
+
+    private List<GamePerson> names;
+    private List<Line> lines = new ArrayList<>();
+    private int ladderHeight;
+
+    public LadderGame(String namesValue) {
+        this(namesValue,1);
+    }
+
+    public LadderGame(String namesValue, int ladderHeight) {
+        this.ladderHeight = ladderHeight;
+        names = Arrays.stream(StringSplit.splitWithDelimiter(namesValue))
+                .map(GamePerson::new)
+                .collect(Collectors.toList());
+    }
```

Comment:
```suggestion
    private LadderGame(List<GamePerson> names, int ladderHeight) {
        this.names = names;
        this.ladderHeight = ladderHeight;
    }

    public static LadderGame of(String namesValue, int ladderHeight) {
        return new LadderGame(
            Arrays.stream(StringSplit.splitWithDelimiter(namesValue))
                    .map(GamePerson::new)
                    .collect(Collectors.toList()),
           ladderHeight         
        );
    }
```

생성자에게는 생성자의 역할만 부여해주세요!
그 외에 필요한 로직은 of 같은 메소드를 만들어 사용하면 된답니다~